### PR TITLE
Graffinprogres

### PR DESCRIPTION
**Pull request by Jarmister**
chart js bez użycia localstorage

### CODE REVIEW

File: src/components/Chart.js [hunk 1]
```diff
@@ -1,5 +1,64 @@
...
+    value: 2
+  },
+  {
+    date: "some_date_3",
+    value: 3
+  }
+];
+
+class Chart extends Component {
+  constructor(props) {
```
Comment (oneacik) [inline src/components/Chart.js:7]: This Chart should be used like this:
`<Chart values={variableContainingDatesAndValues}>`

These values^ will be then passed as props.values.
Please read more about props in react and improve this code.

Example:
`const values = [[new Date(), 20.20],[new Date(), 21.20]]`

File: src/components/Chart.js [hunk 2]
```diff
@@ -1,5 +1,64 @@
-import React from "react";
+import React, { Component } from "react";
+import ChartComponent, { Line } from "react-chartjs-2";
 
-const Chart = () => <div>Chart Component</div>;
+let values = [
+  {
+    date: "some_date_1",
```
Comment (oneacik) [inline src/components/Chart.js]: Date should use this object:
https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Date

File: src/components/Chart.js [hunk 3]
```diff
@@ -1,5 +1,64 @@
...
+    date: "some_date_2",
+    value: 2
+  },
+  {
+    date: "some_date_3",
+    value: 3
+  }
+];
+
+class Chart extends Component {
```
Comment (oneacik) [inline src/components/Chart.js:6]: When you change date to real dates, then please check how to make this graph XY.
Tip. Date can be converted to numeral value.

File: src/components/Chart.js [hunk 4]
```diff
@@ -1,5 +1,64 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      chartData: {
```
Comment (oneacik) [inline src/components/Chart.js:15]: could you attach a link to documentation for chartData?
What are those fields and why is there so many backgroundColor?

Maybe some screenshot too :).

Comment (Jarmister) [reply src/components/Chart.js:15]: https://www.chartjs.org/docs/latest/
  I deleted  unnecessary background colors

File: src/components/Chart.js [hunk 5]
```diff
@@ -1,5 +1,51 @@
...
+
+
+class Chart extends Component {
+  constructor(props) {
+    super(props);
+    let array = props.values;
+    let dates = array.map(([date,])=>date),
+        values = array.map(([,value])=>value);
+
+    let seconds = dates.map(date=> Math.floor(new Date().getTime()/1000))
```
Comment (oneacik) [inline src/components/Chart.js:13]: you can create a function getSecondsFromDate()

**Comment by oneacik** [comment]
Nice.

**Review by oneacik — APPROVED** [review]
Resolve conflicts merging in master and you are free to go :)
just this date function

**Pull request merged**